Components/ProductQuery/RemoteProductQuery.php

```diff
@@ -109,6 +124,15 @@ protected function getConnectProduct($row)
         $product = new Product(
             $row
         );
+
+        $this->eventManager->notify(
+            'Connect_Supplier_Get_Single_Product_Before',
```

Comment:
@sbofirov so it makes only sense to implement it for the localProductGateway or not? In the RemoteProductQuery it currently don't make sense to implement an event here. Because for the insert of products from supplier to merchant there is already a event https://docs.enterprise.shopware.com/connect/connect_extend_plugin/ "Connect_Merchant_Saving_ArticleAttribute_Before"